training/ilab-wrapper/ilab

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# HF caching uses relative symlink structures, so keep cache relative to
+# the central working directory
+CONTAINER_CACHE="/instructlab/cache"
+HOST_CACHE="$(pwd)/cache"
+WORKDIR="$(pwd)"
+mkdir -p "$HOST_CACHE"
+PODMAN_COMMAND=("podman" "run" "--rm" "-it" "--device" "nvidia.com/gpu=all" \
+		"--security-opt" "label=disable" "--net" "host" \
+		"-v" "$WORKDIR:/instructlab" "--entrypoint" "" \
+		"-e" "HF_HOME=$CONTAINER_CACHE" \
+		"quay.io/ai-lab/instructlab-nvidia:latest")
```

Comment:
Need a way to make this configurable so that the script can be shared.  Perhaps sed the 

sed -s/VENDOR/${VENDOR}/g /usr/local/bin/ilab

Reply:
Agree I’ll update this to be more of a template